src/twisted/python/failure.py

```diff
@@ -337,7 +343,9 @@ def __init__(self, exc_value=None, exc_type=None, exc_tb=None, captureVars=False
                 tb = self.value.__traceback__
 
         frames = self.frames = []
-        stack = self.stack = []
+
+        # Backwards compatibility, not used anymore:
+        self.stack = []
```

Comment:
Maybe make it a property and raise a Deprecaton warning each time it is read or written?